src/sdk/pynni/nni/platform/local.py

```diff
@@ -35,10 +36,28 @@
 _log_file_path = os.path.join(_outputdir, 'trial.log')
 init_logger(_log_file_path)
 
+_param_index = 0
+
+def _send_request_parameter_metric():
```

Comment:
Suggest to rename to request_next_parameter()

Reply:
Thanks, renamed it per comments.